packages/browser-wallet/src/popup/shell/Routes.tsx

```diff
@@ -144,6 +145,10 @@ export default function Routes() {
                     element={<AddAccount />}
                     path={`${relativeRoutes.home.account.path}/${relativeRoutes.home.account.add.path}/*`}
                 />
+                <Route
+                    element={<AddTokens />}
+                    path={`${relativeRoutes.home.account.path}/${relativeRoutes.home.account.tokens.path}/${relativeRoutes.home.account.tokens.add.path}/*`}
```

Comment:
I feel like this is a slippery slope.. couldn't we make a helper function to give a relative path, given a base path and an absolute path?

```ts
relativePath(from RoutePath, to: RoutePath): string; // f.x. relativePath(absoluteRoutes.home.account, absoluteRoutes.home.account.tokens.add)
```